docs/fsharp/language-reference/structures.md

```diff
@@ -42,9 +42,51 @@ The following code examples illustrate structure definitions.
 
 [!code-fsharp[Main](../../../samples/snippets/fsharp/lang-ref-1/snippet2501.fs)]
 
+## ByRefLike structs
+
+You can define your own structs that can adhere to `byref`-like semantics (see [Byrefs](byrefs.md) for more). This is done with the <xref:System.Runtime.CompilerServices.IsByRefLikeAttribute> attribute:
```

Comment:
for more --> for more information
rather than include the cross-reference in parens, add a semicolon after sementics and remove the parens.